cloud/pkg/cloudhub/handler/message_handler.go

```diff
@@ -132,6 +138,10 @@ func (mh *messageHandler) HandleConnection(connection conn.Connection) {
 		// add node session to the session manager
 		mh.SessionManager.AddSession(nodeSession)
 
+		err := updateAnnotation(context.Background(), nodeID)
```

Comment:
Will it be better to update the annotations in edgecontroller/upstream? When updates failed, it can be retry.